### PR TITLE
Updated code for issues #6 ,#8

### PR DESCRIPTION
**Pull request by rahlvers**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -111,6 +113,18 @@
             method: 'POST',
             json: cred
         }, function (err, res) {
+            if(err) {
+
+                if(err.code == 'ENOTFOUND') {
```
Comment (anandnagraj) [inline app.js]: Please add a block for ETIMEDOUT as well. The else block will suffice for all other error codes. These 3 seem to be relevant for a client connection.

**Comment by anandnagraj** [comment]
Currently, when a non administrator user tries to access account details, it says no properties and does not show contents. This is fine since the user does not have privilege to access account details. In addition to this behavior I feel that the error code sent by the server should also be propagated to the UI similar to the fix done for "error connection refused". Doing this will also cover other error scenarios while accessing a resource.

File: app.js [hunk 2]
```diff
@@ -111,6 +113,21 @@
             method: 'POST',
             json: cred
         }, function (err, res) {
+            if(err) {
+
```
Comment (anandnagraj) [inline app.js]: Unnecessary blank line

File: views/src/app/home/shared/home.service.ts [hunk 3]
```diff
@@ -98,4 +98,9 @@ export class HomeService {
         }))
           .catch((error:any) => Observable.throw(error.json().error || 'Server error'));
     }
+    deleteSession() {
```
Comment (anandnagraj) [inline views/src/app/home/shared/home.service.ts]: I still see the following issues: 
1. Upon logging out, the session is not deleted.
2. Upon Remove service operation, the session is not deleted.
3. When the browser is closed, the session is not deleted. 
4. The delete icon on the session is no longer seen to invoke the delete operation manually.

I did login with the newer credentials successfully.

Comment (rahlvers) [reply views/src/app/home/shared/home.service.ts]: I am also seeing the delete issue.

**Comment by rahlvers** [comment]
Verified: 
Fixes start vs build/start  - closes #13 
closes #6 
closes #12 
closes #8 

Logo still not changed.  Session issue still not resolved.

**Comment by rahlvers** [comment]
Approved in SSM TWG 02/06/08 for subset of fixes.

**Pull request merged**